W-16903096-Avro-support-in-Studio-CP (#569)
mulesoft/docs-studio: modules/ROOT/pages/import-api-specification-exchange.adoc

. To select the version of the API to import, use the *Selected Modules* table.
. Click *Finish*.

> Studio scaffolds your API spec. If you choose not to create flows out of your API specification, unselect *Scaffold Flows From These API Specifications*.

Reviewer: ```suggestion
By default, Studio scaffolds your API spec. To prevent Studio from creating flows out of your API spec, unselect *Scaffold Flows From These API Specifications*.
```

Author: OK! Thanks!

You can select the version of the API to import in the *Selected Modules* table.
. Click *Finish*.

> Studio scaffolds your API spec. If you choose not to create flows out of your API spec, unselect *Scaffold Flows From These API Specifications*.

Reviewer: ```suggestion
By default, Studio scaffolds your API spec. To prevent Studio from creating flows out of your API spec, unselect *Scaffold Flows From These API Specifications*.
```

Author: OK! Thanks!
